utility/doc/E2E_FEATURE_PATH_TEMPLATE.md

```diff
@@ -186,6 +196,19 @@ This is the 🥩 (or 🥙) of the whole feature. In several PRs, you will implem
 - Additions / modifications to the [E2E testing framework](../../e2e)
 - Additions / modifications to the documentation, code structure and diagrams
 
+### Deliverables
```

Comment:
Lines 92 through 111 contain the deliverables that the author of these github issues would copy paste when create a new issue. This way, it'll be directly in the actual ticket when we assign it to someone (on protocol, on backend, externally, etc...).

Do you think we should merge the two into one or keep this as a separate list of deliverables?

Reply:
Great point. Not sure. As you pointed out in the previous comment, the deliverables listed by this PR are possibly too specific to trustless relays work, whereas the ones on Lines 92 through 111 are very reasonable but also very generic.

Perhaps we can even keep the original deliverables, i.e. Lines 92 - 111, and use this new list and/or example PR as one way these deliverables can be fullfilled?

Reply:
@adshmh I'll leave the call to you because I'm very biased by the context I have (as hard as I try not to) regarding what a new member (internal or external) of the protocol team needs. Try to think what YOU would have needed updated to make it clear and obvious

1. I think having two `Deliverables` sections is not great (confusing, extra, reading, etc...)?
2. What do you think would have benefited you more?

Reply:
Thank you for the clarification. I have updated the docs with what I think could help more.